<launch>
	<node pkg="openfabmap2" type="run_node" name="run_node" output="screen">
		<remap from="image" to="/camera/rgb/image_raw"/>
		<param name="vocab" value="$(find openfabmap2)/codebooks/new_ORB_32/vocab.yml"/>
		<param name="clTree" value="$(find openfabmap2)/codebooks/new_ORB_32/clTree.yml"/>
		<param name="trainbows" value="$(find openfabmap2)/codebooks/new_ORB_32/trainbows.yml"/>
		<param name="DetectorType" value="ORB"/>
		<param name="DescriptorType" value="ORB"/>
		<param name="FabMapVersion" value="FABMAP2"/>
		<param name="NewPlaceMethod" value="Sampled"/>
		<param name="sampleRate" value="100"/>
		<param name="visualise" value="true"/>
		<param name="minMatchValue" value="0.98"/>
		<param name="maxMatches" value="1"/>
		<param name="DisableSelfMatch" value="false"/>
		<param name="SelfMatchWindow" value="1"/>
		<param name="DisableUnknownMatch" value="false"/>
		<param name="AddOnlyNewPlaces" value="true"/>
	</node>
</launch>
